Lib/test/test_funcattrs.py

```diff
@@ -478,6 +479,27 @@ def test_builtin__qualname__(self):
         self.assertEqual([1, 2, 3].append.__qualname__, 'list.append')
         self.assertEqual({'foo': 'bar'}.pop.__qualname__, 'dict.pop')
 
+    @support.cpython_only   # See gh-58211
+    def test_builtin__self__(self):
+        import time
+        import builtins
```

Comment:
Are those imports local for testing or can they be made global? btw, whether they are made global or not, can you order them? (import builtins before time)

Reply:
They can be global. I only copied the style from the test before this new one.

Reply:
Fine to keep them local but just for readability, I would swap the imports so that they are alphabetically sorted.